nullaway/src/main/java/com/uber/nullaway/generics/GenericsChecks.java

```diff
@@ -682,7 +762,8 @@ public static Nullness getGenericMethodReturnTypeNullness(
   /**
    * Computes the nullness of the return of a generic method at an invocation, in the context of the
    * declared type of its receiver argument. If the return type is a type variable, its nullness
-   * depends on the nullability of the corresponding type parameter in the receiver's type.
+   * depends on the nullability of the corresponding type parameter in the receiver's type or the
+   * type argument of the method call.
```

Comment:
Eventually we should handle generic instance methods of generic types, e.g.:
```java
class Foo<T extends @Nullable Object> {
  public void <U> T test(U u) { ... }
}
...
Foo<@Nullable String> f = new Foo<@Nullable String>();
String s = f.<Integer>(3);
// Here s is @Nullable
```
But it doesn't have to be part of this pull request.  I don't think the code below will work for this case.

Reply:
Should I still add the test case?

Reply:
Sure, please add it.  If it doesn't work, and it's not easy to fix, just `@Ignore` the test and we can fix in a follow-up